### DISCUSSION TITLE
the beaver is barely visible anymore on splash and about screen

### DISCUSSION THREAD

**Discussion by e4711s** [dbeaver/dbeaver#11600] [General]
I know, this is cosmetic and subjective, but I think the beaver mascot could be featured a bit more prominently again on the splash screen and about dialog. ;)

**Comment by e4711s** [comment]
...much better now - thanks a lot :) 
![image](https://user-images.githubusercontent.com/31035879/157030816-f800e7a4-d66c-4d0e-8cc0-ab29f9c6c4a4.png)

**Reply by spdrcd** [reply]
the one in the middle.... is that like a parody of github's octocat?
https://github.com/octocat and postgres elephant?

**Reply by LonwoLonwo** [reply]
Maybe :)
